src/frontend/Komponenter/Behandling/GamleBehandlinger/GamleBehandlinger.tsx

```diff
@@ -0,0 +1,85 @@
+import React, { useEffect, useState } from 'react';
+import { useApp } from '../../../App/context/AppContext';
+import DataViewer from '../../../Felles/DataViewer/DataViewer';
+import { byggTomRessurs, Ressurs } from '../../../App/typer/ressurs';
+import { GammelBehandling } from '../../../App/typer/fagsak';
+import styled from 'styled-components';
+import { Link } from 'react-router-dom';
+import { Table } from '@navikt/ds-react';
+import { formaterIsoDato } from '../../../App/utils/formatter';
+import { formatterEnumVerdi } from '../../../App/utils/utils';
+
+const StyledGamleBehandlinger = styled.div`
+    width: inherit;
+`;
+
+const GamleBehandlinger = () => {
+    const { axiosRequest } = useApp();
+    const [gamleBehandlinger, settGamleBehandlinger] = useState<Ressurs<GammelBehandling[]>>(
+        byggTomRessurs()
+    );
+
+    useEffect(() => {
+        axiosRequest<GammelBehandling[], null>({
+            method: 'GET',
+            url: `/familie-ef-sak/api/behandling/gamlebehandlinger`,
+        }).then((res: Ressurs<GammelBehandling[]>) => settGamleBehandlinger(res));
+    }, [axiosRequest]);
+
+    return (
+        <DataViewer response={{ gamleBehandlinger }}>
+            {({ gamleBehandlinger }) => (
+                <StyledGamleBehandlinger>
+                    <h1>Gamle behandlinger</h1>
+                    <>
+                        <Table size="medium">
```

Comment:
Hvis det blir mange rader kan det være fint med zebra? 
```suggestion
                        <Table size="medium" zebraStripes={true}>
```